act-rules-format.bs

```diff
@@ -266,6 +264,8 @@ Some input aspects are well defined in a formal specification, such as HTTP mess
   </ul>
 </div>
 
+[=Atomic rules=] MUST list the aspects used as input for the [applicability](#applicability-atomic) and [expectations](#expectations-atomic) of the atomic rule. Rules can operate on several aspects simultaneously, such as both the HTTP messages and the DOM tree. The method through which an input aspect is served is not relevant. For example a DOM tree can be served through HTTP as HTML, it can be bundled as several pages in an [EPUB publication](http://www.idpf.org/epub3/latest/), or it can be inferred from a [JSX source file](https://facebook.github.io/jsx/). All rules that have only DOM tree as an input aspect can be applied to those technologies.
```

Comment:
I think it would be relevant to move the MUST part up above the examples, so that requirements come first, and examples and clarification for implementation comes later.
Suggestion:  

### Input Aspects (Atomic rules only) ### {#input-aspects}

An input aspect is a distinct part of the [=test subject=]. Rendering a particular piece of content to an end user commonly involves different technologies, some or all of which are required as input for [=atomic rule=]. For example, some rules need to operate directly on the [Hypertext Transfer Protocol](https://tools.ietf.org/html/rfc7230) [[http11]] messages exchanged between a server and a client, while others need to operate on the [Document Object Model](https://dom.spec.whatwg.org) [[DOM]] tree exposed by a web browser.

MOVED: [=Atomic rules=] MUST list the aspects used as input for the [applicability](#applicability-atomic) and [expectations](#expectations-atomic) of the atomic rule. 

MOVED: Rules can operate on several aspects simultaneously, such as both the HTTP messages and the DOM tree. 

Some input aspects are well defined in a formal specification, such as HTTP messages,the DOM tree, and CSS styling [[css-2018]]. For these, a reference to the corresponding section in the [Common Input Aspects note](https://w3c.github.io/wcag-act/NOTE-act-rules-common-aspects.html) is sufficient as a description of the aspect. For input aspects that are not well defined, an ACT Rule MUST include either a detailed description of the aspect in question, or a reference to a well defined description.

<div class=example>
  <p>**Example:** Input aspects for a rule that checks if a transcript is available for videos:</p>
  <ul>
    <li>DOM Tree</li>
    <li>CSS Styling</li>
    <li>Audio output</li>
    <li>Visual output</li>
  </ul>
</div>

<div class=example>
  <p>**Example:** Input aspects for a rule that checks for use of (language specific) generic link texts like "click here" and "more":</p>
  <ul>
    <li>DOM Tree</li>
    <li>CSS Styling</li>
    <li>Language</li>
  </ul>
</div>

The method through which an input aspect is served is not relevant. For example a DOM tree can be served through HTTP as HTML, it can be bundled as several pages in an [EPUB publication](http://www.idpf.org/epub3/latest/), or it can be inferred from a [JSX source file](https://facebook.github.io/jsx/). All rules that have only DOM tree as an input aspect can be applied to those technologies.

Reply:
@WilcoFiers @annethyme I've move the two sentences up as suggested.  Does that mean this can now be merged?